app/cms-client/src/components/Pages/Foo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Page = () => (
+  <div>
+    This is a Foo page
+    <Link to="/admin/innovation">Back</Link>
```

Comment:
Links like this probably need to use `withRouter` to figure out what their root URL is.